### ISSUE TITLE
Right to Left

### ISSUE THREAD

**Issue by ZahraAsgharzadeh** [Akhilendra/sidebarAppiOS#1]
Hi , I've used your project , But I have a problem , I want the menu appears from right to left not left to right . Could you please answer me how to fix this problem ?!? 🤓

**Comment by Akhilendra** [comment]
You just need to add sidebar to right when you add it as subview. Then
change it's x position as needed.

**Comment by ZahraAsgharzadeh** [comment]
self.navigationController?.view.addSubview(sidebarView)
 you're mean is here ?
how can I set it to right . I didn't understand exactly dear friend . 🤯

**Comment by Fatemeh1994** [comment]
Oh , I have this problem too . Help please .Help please

**Comment by ZahraAsgharzadeh** [comment]
Could you please answer me ?
I'm waiting for your answer dear friend 🙂

**Comment by Akhilendra** [comment]
No. Where you set the frame of the sidebar. Change the x position.

**Comment by Akhilendra** [comment]
Was your issue resolved?

**Comment by ZahraAsgharzadeh** [comment]
Dear friend , My new issue is that it comes from center to the right , But I want it to come from right side of the screen to center 🤔

**Comment by ZahraAsgharzadeh** [comment]
Sorry It resolved , I just increase the mount of X to 500 . Thank you 🌹

**Comment by ZahraAsgharzadeh** [comment]
Sorry , I face new issue 🤯😣
When I change the mount of X , It should be different in each device . 
How can I set that , to be the same size in each iPhone device ???
🧐

**Comment by sanayasmeen484** [comment]
How did you manage that black screen to come only on half page

**Comment by Akhilendra** [comment]
The width of the black screen is set to screen width - sidebar width and it's x position is set to sidebar max x position.
